isaric/parsers/isaric-ecmo.toml

```diff
@@ -760,13 +740,18 @@
   fields = [
     { field = "oxygen_cmoccur", values = { 1 = true, 2 = false }, description = "Any oxygen therapy?" },
     { field = "daily_fi02_lbyn", ref = "Y/N/NK", description = "Any supplemental oxygen?" },
+    { field = "invasive_proccur", values = { 1 = true, 2 = false }, description = "Invasive ventilation?" },
+    { field = "daily_invasive_prtrt", values = { 1 = true, 2 = false }, description = "Invasive ventilation?" },
+    { field = "nasal_highflow_proccur", ref = "Y/N/NK" },
+    { field = "daily_nasaloxy_cmtrt", ref = "Y/N/NK" },
+    { field = "noninvasive_proccur", values = { 1 = true, 2 = false }, description = "Noninvasive ventilation?" },
+    { field = "daily_noninvasive_prtrt", values = { 1 = true, 2 = false }, description = "Noninvasive ventilation?" },
+    { field = "extracorp_prtrt", ref = "Y/N/NK" },
+    { field = "daily_ecmo_prtrt", ref = "Y/N/NK" },
+    { field = "pronevent_prtrt", description = "Ventilation in prone position?", ref = "Y/N/NK" },
```

Comment:
@pipliggins the CRF label for this variable indicates Prone positioning, although the variable name suggests prone ventilation. The follow up duration question also refers to prone positioning, so perhaps it is best to not include this variable in the oxygen_therapy attribute